resources/elk6/pipeline/1000_nessus_process_file.conf

```diff
@@ -52,32 +52,6 @@ filter {
       convert => { "risk_number" => "integer"}
       convert => { "total_times_detected" => "integer"}
     }
-
-    if [cvss] == 0 {
```

Comment:
Having moved the CVSS mapping to vulnwhisperer is great! Thanks for the hard work :)